src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -169,6 +193,7 @@ class ShareView extends React.PureComponent {
   setMixedView = () => this.setState({ showEditor: true, showPreview: true });
 
   setDefaultModule = id => this.setState({ defaultModule: id });
+  clearDefaultModule = () => this.setState({ defaultModule: null });
```

Comment:
I think that it's more clear for the user if we select the main module by default, and don't append `module=blah` to the url if the main module is selected. What do you think?